src/sentry/integrations/pagerduty/utils.py

```diff
@@ -116,6 +123,18 @@ def build_incident_attachment(
     }
 
 
+def attach_custom_severity(data: dict[str, Any], action: AlertRuleTriggerAction) -> dict[str, Any]:
+    # use custom severity (overrides default in build_incident_attachment)
+    if action.sentry_app_config is None:
```

Comment:
nit: this can just be `if not action.sentry_app_config` and same on line 132 it can be `if severity`